internal/node/workload_mgr.go

```diff
@@ -293,15 +293,39 @@ func (w *WorkloadManager) RunningWorkloads() ([]controlapi.MachineSummary, error
 	summaries := make([]controlapi.MachineSummary, len(procs))
 
 	for i, p := range procs {
+		uptimeFriendly := "unknown"
```

Comment:
Might be good to break this out into an `enrich()` method... someday.

Reply:
there's already `myUptime`, but since I copied that from NATS server I didn't want to mess with it... yet.